.istanbul.yml

```diff
@@ -1,8 +1,11 @@
 instrumentation:
   root: ./src
-  extensions: 
+  extensions:
     - ".js"
     - ".jsx"
-  excludes: 
-    - "**/node_modules/**"
-    - "**/__tests__/**"
+check:
```

Comment:
Do we want these up to 100%? Try it, if it passes / isn't too much work that's the standard we want on all new projects.